.github/actions/integration-tests/run-file/oslat-all-in-one.json

```diff
@@ -0,0 +1,46 @@
+{
+    "mv-params": {
+        "global-options": [
+            {
+                "name": "common-params",
+                "params": [
+                    { "arg": "duration", "vals": [ "10" ], "role": "client" },
+                    { "arg": "rtprio", "vals": [ "1" ], "role": "client" }
+                ]
+            }
+        ],
+        "sets": [
+            {
+                "include": "common-params"
+            }
+        ]
+    },
+    "tool-params": [
+        {
+            "tool": "sysstat",
+            "params": [
+                { "arg": "subtools", "val": "mpstat", "enabled": "yes" }
+            ]
+        }
+    ],
+    "tags": {
+        "run": "single-json-all-in-one",
+        "repeat": "CI_REPEAT_RUNS",
+        "userenv": "USERENV"
+    },
+    "endpoint": {
+        "k8s": "",
+        "controller-ip": "CONTROLLER_IP",
+        "host": "CI_ENDPOINT_HOST",
+        "user": "CI_ENDPOINT_USER",
+        "userenv": "USERENV",
+        "unique-project": "1",
+        "kubeconfig": "0",
+        "client": "1-2",
+        "server": "1",
+        "cpu-partitioning": "default:1",
+        "securityContext": "client-1:SCRIPT_DIR/k8s-endpoint/securityContext-oslat.json",
+        "securityContext": "client-2:SCRIPT_DIR/k8s-endpoint/securityContext-oslat.json"
```

Comment:
You can't have 2 identical keys...the 2nd one overrides the first one...so `client-1` is not getting the security context it requires to set SCHED_FIFO!

Reply:
I don't remember defining unique keys in the schema... for this test, I don't need 2 clients, but we need to support this somehow. What would be the correct form then?

Reply:
I think we should consider making the values capable of being a string (for single value) or an array of strings (for multi-value).

We are going to have to be a bit creative here, from the perspective of Crucible/Rickshaw having multiple "keys" with the same "value" is totally fine.  However, obviously from the perspective of JSON that isn't ok...

Reply:
Actually, we shouldn't be too beholden to our original format...maybe we should allow for something like this:

```
...
  "key1": "value1",
  "key2": [
    {
      "scope": "scope2.1",
      "value": "value2.1"
    },
    {
      "scope": "scope2.2",
      "value": "value2.2"
    }
  ]
...
```

With the advanced data structure capability that JSON provides there really is no reason for us to be settling on a format that requires us to parse out the scope and value -- and in fact it is limiting because then it means that whatever the separator character is cannot be used in the values for each field.

Thoughts on above @atheurer ?

Reply:
I don't see how we build the args stream without parsing the key:value pairs individually.

If you have a list like:
```
>>> json={"SecurityContext": ["client-1:foo","client-2:bar"]}
>>> json["SecurityContext"]
['client-1:foo', 'client-2:bar']
```
You need to append "--endpoint SecurityContext:client-1:foo,client-2:bar", so you need to get the values one by one. So you need to iterate over all the keys and values...

On the other side, if you have a simple string like:
```
>>> json={"foo": "bar"}
>>> json["foo"]
'bar'
```
You need to append "--endpoint foo:bar", which means you only need to get the key "foo" and the value json["foo"].

That's why I think parsing is necessary... and determining if it is a list or string is also required:
`isinstance(var, str)`
or
`isinstance(var, list)`

Am I missing anything ?

Reply:
I was thinking that you would
just iterate over the list and add the the key:scope:value for each instance of scope:value.

It's kinda hard for me to illustrate this on my phone 😞

Reply:
> I was thinking that you would just iterate over the list and add the the key:scope:value for each instance of scope:value.
> 
> It's kinda hard for me to illustrate this on my phone disappointed

Ok but if we have a key:value and key:[value,value] we need to handle them differently:
```
  "key1": "value1",
  "key2": [
```

Unless you are suggesting to always have the same format with `key:scope:value`